data/SandipPalit.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Sandip Palit",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " **Data Engineer** | Data Ecience Explorer | Kaggle Expert | Speaker | Loading..",
```

Did you mean "Data Science" here!? 🤔

```suggestion
  "bio": " **Data Engineer** | Data Science Explorer | Kaggle Expert | Speaker | Loading..",
```

---

Hey, 

Typo fixes can be in another PR. It's not a blocker.